src/napari_spatialdata/_interactive.py

```diff
@@ -232,11 +234,14 @@ def _add_image(self, key: str) -> None:
         selected_cs = self.coordinate_system_widget._system
         img = self._sdata.images[key]
         affine = _get_transform(self._sdata.images[key], selected_cs)
+        new_image, rgb = _transform_to_rgb(element=self._sdata.images[key])
```

Comment:
minor nitpick: I don't find this variable name to be very clear. Perhaps use the same pattern as above in the add label method (i.e., `rgb_image`)

Reply:
Adressed with [b43f9b2](https://github.com/scverse/napari-spatialdata/pull/115/commits/b43f9b232f6486d5dba740dc2342a86c9de768a0)